cmd/crc-embedder/cmd/embed.go

```diff
@@ -21,13 +22,31 @@ import (
 var (
 	goos       string
 	cacheDir   string
+	components []string
 	noDownload bool
 )
 
+const (
+	vfkitDriver      = "vfkit-driver"
+	vfkitEntitlement = "vfkit-entitlement"
+	libvirtDriver    = "libvirt-driver"
+	adminHelper      = "admin-helper"
+	tray             = "tray"
+)
+
+var componentNames = []string{
+	vfkitDriver,
+	vfkitEntitlement,
+	libvirtDriver,
+	adminHelper,
+	tray,
+}
+
```

Comment:
Might be more extensible in the future to build the list of allowed components directly from the `dataFileUrls` structure? This would allow you to take into account `goos` when showing the list of allowed components
```
func allComponentNames() []string {
  var components []string
  for _, dls := range datafileUrls {
    for component, _ := range dls {
        components = append(components, component)
    }
  }
  return components
}
```

and something similar but limited to the current goos